.github/workflows/node-linux.sh

```diff
@@ -63,14 +63,20 @@ done
 # Start keploy in test mode.
 sudo -E env PATH=$PATH ./../../keployv2 test -c 'node src/app.js' --delay 10
 
```

Comment:
instead of node src/app.js you can replace it with ```npm start --prefix /src/app.js`` in both test and record

Reply:
done!